recipes-core/images/asteroid-image.bbappend

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Just remove that extra empty line and it's good to go (don't forget to update asteroidos.org)